sdk/eventhubs/azure_messaging_eventhubs/Cargo.toml

```diff
@@ -40,6 +40,7 @@ azure_messaging_eventhubs = { path = ".", features = [
   "test",
 ] }
 criterion.workspace = true
+dotenvy = "0.15.7"
```

Comment:
I'd rather not spread out this dependency all over. Rather than this and the changes in `benchmarks.rs` below, can you make `azure_core_test::find_ancestor_file` public (maybe rename it to `find_ancestor_path` while you're at it) and call that? Remove the `cfg` since it'll be public now. I just added that since we didn't call it from wasm32 code and it was causing build warnings-as-errors.

Reply:
Absolutely -  I can add a new public function which calls `dotenvy::from_path` and `find_ancestor_file`, and then update the recorded test start to use the new function.